packages/composites/lido/src/endpoint/exchangerate.ts

```diff
@@ -0,0 +1,38 @@
+import { Validator, Requester } from '@chainlink/ea-bootstrap'
+import { AdapterContext, AdapterRequest, AdapterResponse } from '@chainlink/types'
+import { makeExecute as makeViewFunction } from '../../../../sources/view-function'
```

Comment:
This should be added as a dependency in `package.json` rather out of the mono-repo.